doc/sphinx/yaml/phases.rst

```diff
@@ -0,0 +1,720 @@
+.. highlight:: yaml
```

Comment:
Is there any way to auto-generate these files? Or at least the sections with the documentation of the specific options?

Reply:
I don't think so -- without these files, where would this documentation come from?